crypto/argon2/argon2.c

```diff
@@ -0,0 +1,195 @@
+/*
+ * Argon2 reference source code package - reference C implementations
```

Comment:
Does this comply with:

https://datatracker.ietf.org/doc/draft-irtf-cfrg-argon2/

I note that document is still in draft (and recently expired) and is not standards track. We can't merge this until that is published (or we're pretty sure its going to be). I'm not sure what our position on Informational RFCs is.

Reply:
It was actually updated a few days ago.

Reply:
I mean a month ago

Reply:
It does indeed. I have compared it with the draft version that used in reference implementation and changes therein do not affect implementation as such.